query_prototype/views.py

```diff
@@ -47,6 +51,84 @@ def prototype_ui(request):
 
             return render(request, "query_blank.html", view_variables)
 
+def get_mock_results():
+
+    modified_filename = "{0}{1}query_prototype{1}modified_query_results.json".format(os.getcwd(), os.sep)
+    with open(modified_filename, "r") as input_file:
+        query_results = json.loads(input_file.read())
+
+    return query_results
+
+def get_manual_results():
+
+    # a. Prepare data for query
+    # (NOTE: This should be done in future in form data cleaning function)
```

Comment:
TODO works better here because that gets parsed by the IDE (e.g. [here](https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight)). Helps to not loose track
```suggestion
    # (TODO: This should be done in future in form data cleaning function)
```